hommod_rest/services/model.py

```diff
@@ -274,6 +274,17 @@ def _set_template(self, tempac, oligomerize=True):
         except Exception as e:
             _log.warn('Cannot execute BuildSymRes on {}: {}'.format(tempac,
                                                                     e.args[0]))
+        # YASARA also cleans when starting a modeling run, but
+        # we need to make sure that we have the molecule in its
+        # final state, before we start reading from it.
+        try:
+            self.yasara.CleanObj(tempobj)
+        except RuntimeError:
```

Comment:
Then no exception is thrown I guess. You're right, this is wrong, should I just throw a default exception here?